packages/react-shared-contexts/README.md

```diff
@@ -0,0 +1,8 @@
+# @fluentui/react-shared-contexts
+
+**Shared contexts for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+([formerly Office UI Fabric React](https://developer.microsoft.com/en-us/office/blogs/ui-fabric-is-evolving-into-fluent-ui/))
```

Comment:
Let's not include this line for new packages (and we should probably remove it from the template)
```suggestion
```

Reply:
Removed.